go.mod

```diff
@@ -73,6 +73,7 @@ require (
 	github.com/weaveworks/flux v0.0.0-20190729133003-c78ccd3706b5
 	github.com/weaveworks/github-release v0.6.2
 	github.com/weaveworks/launcher v0.0.0-20180711153254-f1b2830d4f2d
+	github.com/whilp/git-urls v0.0.0-20160530060445-31bac0d230fa
```

Comment:
You need to bump the dependency cache image

Reply:
Thanks. I'm pushing the image now. It just takes long with my internet connection...